### PR TITLE
Update push-duty docs to mention addons-server locales extraction

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

File: docs/server/push-duty.rst [hunk 1]
```diff
@@ -145,11 +145,14 @@ deployed revision and tag e.g:
...
-+++++++++++++
+Extract locales
++++++++++++++++
 
-After you tag the ``addons-frontend`` repository, please `extract and merge the L10n strings <https://github.com/mozilla/addons-frontend/blob/master/docs/i18n.md>`_.
+Once you are done pushing the tags to stage:
 
+ * Run the ``./bin/run-l10n-extraction`` command in ``addons-frontend`` repository (`documentation <https://github.com/mozilla/addons-frontend/blob/master/docs/i18n.md>`_).
+ * Run the ``./scripts/extract-l10n.sh`` command in ``addons-server`` repository.
+
```
Comment (eviljeff) [inline docs/server/push-duty.rst:155]: Should we mention the differences around git commits here? How they work is quite different - addons-frontend pushes with the current git credentials; addons-server relies on an API key being set(?)

Comment (eviljeff) [reply docs/server/push-duty.rst:155]: 👍

**Review by eviljeff — APPROVED** [review]
r+wc

**Comment by willdurand** [comment]
The build failed here and continues to fail: https://app.circleci.com/pipelines/github/mozilla/addons/40/workflows/011534a1-6b49-4870-83a6-608bb12e859f/jobs/40

**Comment by diox** [comment]
Sigh reStructuredText... I'll fix

**Comment by eviljeff** [comment]
still failing ...

**Pull request merged**